tests/api/state/stateManager.js

```diff
@@ -160,4 +161,19 @@ tape('StateManager', (t) => {
       st.end()
     })
   })
+
+  t.test('should pass Common object when copying the state manager', st => {
+    const stateManager = new StateManager({
+      common: new Common('goerli', 'constantinople')
+    })
+
+    st.equal(stateManager._common.chainName(), 'goerli')
+    st.equal(stateManager._common.hardfork(), 'constantinople')
+
+    const stateManagerCopy = stateManager.copy()
+    st.equal(stateManagerCopy._common.chainName(), 'goerli')
+    st.equal(stateManagerCopy._common.hardfork(), 'constantinople')
+
+    st.end()
+  })
```

Comment:
Some nit-picking: `constantinople` is a HF on Goerli which is just jumped through, see [goerli.json](https://github.com/ethereumjs/ethereumjs-common/blob/master/src/chains/goerli.json) file in the Common lib.

This might be technically correct but I fear the more we use examples like this in the code the more get people confused about this `Constantinople` <-> `Petersburg` relationship (I really hate that we didn't have upgraded the `Constantinople` name to be the fixed one, think this will remain some never ending re-explanation topic).

So I would suggest to use `petersburg` here, which one would also do in a practical context. If you think this is to minor to update let me know, I would otherwise approve, just personally think it would be worth the change.

Reply:
Or maybe `byzantium` so that it's not the same as the default value (`petersburg`)

Reply:
Good point Holger!

I just needed a non-default value. Now that #524 has been merged, I can use `byzantium`.